scripts/unity/header_prepare.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+import sys
+import re
+import argparse
+
+def func_names_from_header(in_file, out_file, out_wrap_file):
+    content = None
+    with open(in_file) as f_in:
+        content = f_in.read()
+
+    if content == None:
+        return
+
+    # change static inline functions to normal function declaration
+    static_inline_pattern = re.compile(r"((__deprecated )?)(static inline )((struct )?)(\w+ )(\*?)((_impl_)?)(\w+\([^\)]+\))(\n\{[\s\S]+?\n\})", re.M | re.S)
+    (content, static_inline_cnt) = static_inline_pattern.subn(r"\4\6\7\10;", content)
```

Comment:
What about these macro's?

INLINE
__STATIC_INLINE
ALWAYS_INLINE

Reply:
Added __STATIC_INLINE and ALWAYS_INLINE, I haven't seen INLINE being used in API. Added also examples of static inline functions to the example test project.